src/ffmpeg.imageio/ffmpeginput.cpp

```diff
@@ -496,6 +504,26 @@ FFmpegInput::open(const std::string& name, ImageSpec& spec)
                               AV_DICT_IGNORE_SUFFIX))) {
         m_spec.attribute(tag->key, tag->value);
     }
+    tag = NULL;
+    if (m_data_stream >= 0) {
+        while ((
+            tag = av_dict_get(m_format_context->streams[m_data_stream]->metadata,
+                              "", tag, AV_DICT_IGNORE_SUFFIX))) {
+            if (strcmp(tag->key, "timecode") == 0) {
+                m_spec.attribute("ffmpeg:timecode", tag->value);
+                break;
+            }
+        }
+    }
+    tag = NULL;
+    while (
+        (tag = av_dict_get(m_format_context->streams[m_video_stream]->metadata,
+                           "", tag, AV_DICT_IGNORE_SUFFIX))) {
+        if (strcmp(tag->key, "timecode") == 0) {
+            m_spec.attribute("ffmpeg:timecode", tag->value);
```

Comment:
I think all the attribute name searches are case-insensitive, but just in case, I think perhaps this should match how it was done for OpenEXR and DPX with as "ffmpeg:TimeCode".